Traffic management concepts corrections (#4481)
istio/istio.io: content/docs/concepts/traffic-management/routing/virtual-services/index.md

## Route requests to services in a Kubernetes namespace {#routing-namespace}

When the `host` field of a route destination in a virtual service is specified using

> a short name such as `svc-1`, Istio will expand it into a fully qualified domain name

Reviewer: ```suggestion
using a short name like `svc-1`, Istio expands the short name into a fully qualified domain name.
```

> by appending a domain suffix based on the namespace of virtual service containing the

Reviewer: ```suggestion
To perform the expansion, Istio adds a domain suffix based on the namespace of the virtual service that contains the
```

> route rule. For example, if the virtual service is defined in namespace `my-namesapce`,

Reviewer: ```suggestion
routing rule. For example, if the virtual service is defined in the `my-namesapce` namespace,
```

> the suffix `my-namespace.svc.cluster.local` will be appended to the destination to form

Reviewer: ```suggestion
Istio adds the `my-namespace.svc.cluster.local` suffix to the abbreviated destination resulting in
```

the actual destination: `svc-1.my-namespace.svc.cluster.local`.

> This approach is very convenient and commonly used to simplify examples, but can

Reviewer: ```suggestion
While this approach is very convenient and commonly used to simplify examples, it can
```

> easily lead to misconfigurations and is therefore

Reviewer: ```suggestion
easily lead to misconfigurations. Therefore, we do
```

[not recommended for production deployments](/docs/reference/config/networking/v1alpha3/virtual-service/#Destination).

The following example shows a virtual service configuration with fully qualified traffic routes

> for two services in the `my-namespace.svc.cluster.local` Kubernetes namespace.

Author: TODO: put this sentence back ... deleted accidentally

    caption="Configurable traffic route based on the namespace of two application services"
    >}}

> Note that this approach is also more flexible in that the two destinations could be

Reviewer: I think I captured the core of what you were saying but I'm not entirely sure.
```suggestion
Using fully qualified hosts in the routing rules provides more flexibility because two destinations in the same namespace can't use the same short name.
```

> in different namespaces but could not be when short names are used.

Reviewer: ```suggestion
```
